oio/blob/auditor.py

```diff
@@ -232,13 +232,17 @@ def __init__(self, conf, **kwargs):
         self.volume = volume
 
     def run(self, *args, **kwargs):
-        while True:
+        work = True
+        while work:
             try:
                 worker = BlobAuditorWorker(self.conf, self.logger, self.volume)
                 worker.audit_pass()
+                work = False
```

Comment:
If an exception occurs without `daemon` mode enabled, `work` still remains `true`, it is by design ? (i.e. we want a full pass)